metaflow/plugins/kfp/tests/.gitlab-ci.yml

```diff
@@ -131,6 +131,7 @@ test:internal:
           export KFP_SDK_NAMESPACE=metaflow-integration-testing-internal && 
           export USER=${GITLAB_USER_EMAIL} &&
           cd /home/zservice/metaflow/metaflow/plugins/kfp/tests &&
+          python -c "import os; print(os.environ[\"METAFLOW_DATASTORE_SYSROOT_S3\"])"
```

Comment:
is this a forgotten print statement?

Reply:
For some reason, I don't see this statement in the diff. Maybe you looked at a previous version? In any case, it's removed from the `.gitlab-ci.yml` file.